Modules/CTP/src/RawDataQcCheck.cxx

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
```

Comment:
It seems that RawDataQcCheck is the default skeleton class. Please remove it.

Reply:
Yes it is. Do you mean removing RawDataQcCheck.cxx and RawDataQcCheck.h ?

Reply:
yes, removing the whole class if you don't use it .

Reply:
also to remove the check in .json file?

Reply:
and line in LinkDef.h?

Reply:
yes to all